components/Header.js

```diff
@@ -47,47 +90,42 @@ class Header extends Component {
           >
             Menu
           </button>
-          <nav
-            className={this.state.menuHidden ? 'hidden main-menu-wrapper' : 'visible main-menu-wrapper'}
-            onClick={this.handleMenuToggle}
```

Comment:
I moved the `handleMenuToggle` `onClick` from this `<nav>` element to the individual links. For accessibility purposes, `onClick` should be on interactive elements, link links. I also added `onKeyDown` to each of those links, as elements should be accessible by both mouse and keyboard.